maestro-test/src/test/kotlin/maestro/test/GraalJsEngineTest.kt

```diff
@@ -0,0 +1,47 @@
+package maestro.test
+
+import com.google.common.truth.Truth
+import com.google.common.truth.Truth.assertThat
+import maestro.js.GraalJsEngine
+import org.graalvm.polyglot.PolyglotException
+import org.graalvm.polyglot.Value
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class GraalJsEngineTest : JsEngineTest() {
```

Comment:
These tests document known behaviors that are different from Rhino JS. Common tests live in the base JsEngineTest class.